paper-toolbar.html

```diff
@@ -227,6 +229,37 @@
         width: auto;
         margin: 0;
       }
+
+      /* TODO(noms): Until we have a better solution for classes that don't use
+       * /deep/ create our own.
+       */
+      .layout.horizontal {
+        @apply(--layout-horizontal);
+      }
+
+      .layout.center {
+        @apply(--layout-center);
+      }
+
+      .layout.start-justified {
+        @apply(--layout-start-justified);
+      }
+
+      .layout.center-justified {
+        @apply(--layout-center-justified);
+      }
+
+      .layout.end-justified {
+        @apply(--layout-end-justified);
+      }
+
+      .layout.around-justified {
+        @apply(--layout-around-justified);
+      }
+
+      .layout.justified {
+        @apply(--layout-justified);
+      }
```

Comment:
awesome! do we still need the CSS for `.layout.justified`, `.layout.around-justified`, and so on...?